logbook-ktor-client/src/main/kotlin/org/zalando/logbook/client/LogbookClient.kt

```diff
@@ -56,12 +57,13 @@ class LogbookClient(
                 val responseProcessingStage = it.call.attributes[responseProcessingStageKey]
                 val response = ClientResponse(it)
                 val responseWritingStage = responseProcessingStage.process(response)
-                if (response.shouldBuffer() && !loggingContent.isClosedForRead) {
-                    val content = loggingContent.readBytes()
-                    response.buffer(content)
+                scope.launch {
```

Comment:
Any reason why we opted to create a coroutine here? Seems it would work fine if we didn't have it here?